src/main/java/com/exercise/tasksmanagementsystem/entity/TaskGroup.java

```diff
@@ -0,0 +1,11 @@
+package com.exercise.tasksmanagementsystem.entity;
+
+public enum TaskGroup {
+
+    TASK_TRACKERS,
+    TASK_FUSION,
+    TASK_UNITY,
+    TASK_FORGE
+
+
```

Comment:
it is more about Clean code: do not leave empty lines, sometimes you can use one for better reading, but two - never :)  but here is no need for any :)